src/IOHprofiler_observer.cpp

```diff
@@ -124,7 +124,7 @@ void IOHprofiler_observer::reset_observer(const IOH_optimization_type optimizati
   if (optimization_type == IOH_optimization_type::Maximization) {
     this->current_best_fitness = std::numeric_limits<double>::lowest();
   } else {
-    this->current_best_fitness = std::numeric_limits<double>::max();
+    this->current_best_fitness = (std::numeric_limits<double>::max)();
```

Comment:
I don't understand the need of parenthesis here, it's not standard, as far as I can tell.

Reply:
This is because mvsc compiler (spef: the windows.h file) has its own macro definition for max and min, which override the function calls and cause syntax errors in this case. The parenthesis prevent his from happening. We can also undefine these macros, but this seemed to be the easier option.